<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>about</class>
 <widget class="QDialog" name="about">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>702</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>关于</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>关于学生选课和课程管理系统</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>761</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文新魏</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>学生选课和课程管理系统

版权所有 2019 ME LLC保留所有权利。

学生选课和课程管理系统的诞生离不开网上QT的教程和相关的代码。

本程序使用了网络的一些图片用于程序的界面美化

如果侵犯了您的权利，请及时与我联系

学生选课和课程管理系统 服务条款：null</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>751</width>
     <height>115</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文新魏</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>学生选课和课程管理系统 已是最新版本（以后也不会更新）

版本 0.0.0.99 (Beta ver)  (64位）

报告问题  请联系462845886@qq.com(联系了也不会获得相关帮助或回复)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_gif">
   <property name="geometry">
    <rect>
     <x>552</x>
     <y>0</y>
     <width>250</width>
     <height>134</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>134</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>134</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文新魏</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>博客：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_blog">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>580</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正姚体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://www.acooz.cn&quot;&gt;&lt;span style=&quot; font-size:16pt; text-decoration: underline; color:#27acff;&quot;&gt;诗酒趁年华&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>630</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文新魏</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GitHub：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_github">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>630</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://github.com/acooz/Course-Management-System&quot;&gt;&lt;span style=&quot; font-size:16pt; text-decoration: underline; color:#27acff;&quot;&gt;Course-Management-System&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>label_gif</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_blog</zorder>
  <zorder>label_6</zorder>
  <zorder>label_github</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
